src/els_config.erl

```diff
@@ -74,8 +74,8 @@ initialize(RootUri, Capabilities, InitOptions) ->
   Config = consult_config(config_paths(RootPath, InitOptions)),
   do_initialize(RootUri, Capabilities, Config).
 
--spec do_initialize(uri(), map(), map()) -> ok.
-do_initialize(RootUri, Capabilities, Config) ->
+-spec do_initialize(uri(), map(), {undefined|path(), map()}) -> ok.
```

Comment:
Things look a bit weird now. The `consult_config/1` function now returns a tuple, but on line 74 we match it as a `Config`. Why do we need this change?

Reply:
The tuple returns the path of the file it was loaded from, so we can return it in the lens result.  `do_initialize` expects the tuple, and stores the path.  So perhaps the variable name needs changing, but it *is* the config, just includes where it came from.